services/blockchain-indexer/shared/jobs/validateValidatorsCache.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { Logger } = require('lisk-service-framework');
+
+const {
+	reloadValidatorCache,
+	getAllValidators,
+} = require('../dataService');
+
+const business = require('../dataService/business');
+
+const logger = Logger();
+
+const validateValidatorCache = async () => {
+	const validatorsByStake = await business.getPosValidatorsByStake({ limit: -1 });
+	// Create a deep copy of cached validators
+	const cachedValidators = (await getAllValidators()).slice();
```

Comment:
May we can use lodash to create deep copy of the cached validators.

Reply:
Applied.